lib/plugins/aws/lib/naming.js

```diff
@@ -374,6 +374,11 @@ module.exports = {
   getStreamConsumerName(functionName, streamName) {
     return `${functionName}${streamName}Consumer`;
   },
+  getServiceSpecificStreamConsumerName(functionName, streamName) {
```

Comment:
it would be good to cover this with unit test

Reply:
Additionally, I think it would be better to not have separate util for that but go with the similar route to `getHttpApiName` for example with encapsulating that logic inside

Reply:
Yes, I'll add tests and will try to make it similar to `getHttpApiName `